MetaMask/metamask-mobile
# feat: Add EIP-6963 Provider

# **Description**

Bump `@metamask/providers` to v13.1.0 which includes EIP6963 support. Pass necessary provider info values to the inpage bridge script to properly initialize the provider.

## **Manual testing steps**

* Visit eip6963.org
* MetaMask Mobile should show up in the list
* Click Connect
* Should show currently connected network

## **Screenshots/Recordings**

<img width="537" alt="Screenshot 2023-10-18 at 4 04 27 PM" src="https://github.com/MetaMask/metamask-mobile/assets/918701/8da4ab0d-fbea-4cdf-ad74-f4bd53366b33">

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/869

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
